tests/Unit/ReportingTest.php

```
@@ -0,0 +1,154 @@
+
+			/** @param mixed ...$args */
+			public function prepare( string $query, ...$args ): string {
+				return $query;
+			}
+
+			/** @return array<int, object> */
+			public function get_results( string $query ): array {
+				return $this->rows;
+			}
+		};
+	}
```

PHPMD (2.15.0)

[error] The method build_wpdb_mock is not named in camelCase. (undefined)

(CamelCaseMethodName)